src/index.js

```diff
@@ -15,15 +18,25 @@ initAnalyticsTracking();
 googleSDK();
 facebookSDK();
 
+const virtualStudies = loadDraftVirtualStudy() || undefined;
+const preloadedState = {
+  virtualStudies,
+};
+const store = initStore(preloadedState);
 
 const render = Component => {
-  ReactDOM.render(<Component />, getAppElement());
+  ReactDOM.render(
+    <Provider store={store}>
```

Comment:
@jberube Provider HOC should only wrap the App Container. There needs to be no state management for static pages.

Reply:
That MaintenancePage don't need a store? :rofl: 

Indeed!